src/ClassicUO.Client/LegionScripting/API.py

```
@@ -0,0 +1,286 @@
+    pass
+
+def Stop():
+    pass
+
+def ToggleAutoLoot():
+    pass
+
+def Virtue(virtue: str):
+    pass
+
+def NearestEntity(scanType: Any, maxDistance: int = 10):
```

Ruff (0.8.2)

Undefined name `Any`

(F821)